inc/options/general.php

```diff
@@ -126,7 +126,7 @@
 		'key' => 'minify.configuration_overloaded'
 	) );
 ?>
-		<p><?php w3tc_e( 'minify.general.header', 'Reduce load time by decreasing the size and number of <acronym title="Cascading Style Sheet">CSS</acronym> and <acronym title="JavaScript">JS</acronym> files. Automatically remove unnecessary data from <acronym title="Cascading Style Sheet">CSS</acronym>, <acronym title="JavaScript">JS</acronym>, feed, page and post <acronym title="Hypertext Markup Language">HTML</acronym>.' ) ?></p>
+		<p><?php w3tc_e( 'minify.general.header', 'Reduce load time by decreasing the size and number of <acronym title="Cascading Style Sheet">CSS</acronym> and <acronym title="JavaScript">JS</acronym> files. Automatically remove unnecessary data from <acronym title="Cascading Style Sheet">CSS</acronym>, <acronym title="JavaScript">JS</acronym>, feed, page and post <acronym title="Hypertext Markup Language">HTML</acronym>.', 'w3-total-cache' ) ?></p>
```

Comment:
Not a translation function.  https://github.com/W3EDGE/w3-total-cache/blob/2.1.5/w3-total-cache-api.php#L601
Use `__()` inside of a `sprintf()` with a `// translators: 1:` comment.

Reply:
Done